Thoms' changes to project managment section (#65)
DanNixon/sine2020_software_guidelines_paper: guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex

If the deadline and time required are similar then it is better to choose a resource who has the relevant skills and experience.
However, the draw back of always choosing the same resource to do similar tasks is that they will become a single point of failure.
If that resource were to leave the team then no-one else would have any knowledge of that part of the project.

> An issue particularly well-known in academia due to the relatively short employment of PhD students and post docs.

Author: ```suggestion
This is a common  issue in academia, due to the relatively short employment of PhD students and post docs.
```

Reviewer: ok

Therefore, it is advisable to use tasks as an opportunity to train less experienced/skilled resources when there is sufficient time between the deadline and the expected duration of the task.

\subsection{Stakeholder Engagement}
\label{se}

> Stakeholders is a general term used to describe people who have some interest in your project (e.g. users, team members and sponsor).

Author: customers are users in this case. From a project management perspective customer is the right word to use here. I think we are better off leaving it as customer and adding a sentence to clarify that the customer is the user

Reviewer: I understand this, but the readers are not project managers and I think that many of them understand the word customer to be somebody who pays for something. However, I have no strong feelings about using one or the other.

Author: ```suggestion
For a software project the users are often considered to be the customers. 
Stakeholders is a general term used to describe people who have some interest in your project (e.g. customers, team members and sponsor).
```

It is up to the individual project to decide what communication method is suitable.

The users are vital sub-group of stakeholders.
An effective way of keeping them engaged is imperative to the success of the project.
One method is to have a committee (this may be called an advisory committee, in the Mantid project it is called the scientific steering committee).
This group will be comprised of the senior project team (i.e. the project manager and any sub-team leaders for a larger project) and a customer from each of the main user communities.
This committee would discuss the progression of tasks and what tasks should be next (priorities).

> User/customer engagement on a frequent basis is an integral part of SCRUM (see Section \ref{Development methodology} where this happen during demonstrations of developed features at the end of each sprint.

Author: this fits in better in the agile/waterfall section not here.

Reviewer: It is about stakeholder managment so I think it fits here. It could also be mention in the agile/waterfall section, but then I think there should be a reference back to the agile/waterfall section in the stakeholder management section.

Author: ```suggestion
The level of stakeholder engagement will depend on the development methodology for the project, see Section \ref{Development methodology}.
```
